Cannot build base testsuite due to missing dependency related to WF

### Describe the bug

The WF distribution is not built by default and the testsuite should be aware of that.

```
[INFO] SAML Adapters Distribution Parent .................. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  09:10 min
[INFO] Finished at: 2022-08-29T06:16:10-04:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project integration-arquillian-tests-base: Could not resolve dependencies for project org.keycloak.testsuite:integration-arquillian-tests-base:jar:999-SNAPSHOT-main-20220829-100527: Could not find artifact org.keycloak:keycloak-wildfly-adduser:jar:999-SNAPSHOT-main-20220829-100527 in central (https://repo.maven.apache.org/maven2) -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :integration-arquillian-tests-base
```

### Version

999

### Expected behavior

_No response_

### Actual behavior

_No response_

### How to Reproduce?

_No response_

### Anything else?

_No response_

--- a/testsuite/integration-arquillian/tests/pom.xml
+++ b/testsuite/integration-arquillian/tests/pom.xml
@@ -2092,10 +2092,6 @@
                     <groupId>org.keycloak</groupId>
                     <artifactId>keycloak-authz-client</artifactId>
                 </dependency>
-                <dependency>
-                    <groupId>org.keycloak</groupId>
-                    <artifactId>keycloak-wildfly-adduser</artifactId>
-                </dependency>
 
                 <!--UNDERTOW-->
 

